### PR TITLE
feat(Bonus Pagamenti Digitali): [#177362328] Update loyalty logo size for privative card & privative wallet card preview

### PR DESCRIPTION
**Pull request by fabriziofff**
## Short description
This pr updates the loyalty logo size for privative card & privative wallet card preview, in order to implements some changes related to ui review.

Schermata 2021-03-16 alle 18 21 39
Schermata 2021-03-16 alle 18 21 49

### CODE REVIEW

**Review by Undermaken — APPROVED** [review]

**Comment by Undermaken** [comment]
@thisisjp wait for your approval

**Comment by fabriziofff** [comment]
@Undermaken we made these changes together with @thisisjp in a live refinement session.

**Pull request merged**
